l1-contracts/test/foundry/l1/unit/concrete/state-transition/data-availability/RollupDAManager.t.sol

```diff
@@ -46,31 +45,33 @@ contract RollupDAManagerTest is Test {
         // Attempt to update DA pair as owner
         vm.startPrank(owner);
         vm.expectEmit(true, true, false, true);
-        emit DAPairUpdated(l1DAValidator1, l2DAValidator1, true);
-        rollupDAManager.updateDAPair(l1DAValidator1, l2DAValidator1, true);
+        emit DAPairUpdated(l1DAValidator1, L2DACommitmentScheme.EMPTY_NO_DA, true);
+        rollupDAManager.updateDAPair(l1DAValidator1, L2DACommitmentScheme.EMPTY_NO_DA, true);
         vm.stopPrank();
 
         // Attempt to update DA pair as non-owner
         vm.startPrank(nonOwner);
         vm.expectRevert("Ownable: caller is not the owner");
-        rollupDAManager.updateDAPair(l1DAValidator2, l2DAValidator2, true);
+        rollupDAManager.updateDAPair(l1DAValidator2, L2DACommitmentScheme.EMPTY_NO_DA, true);
         vm.stopPrank();
     }
 
-    function testUpdateDAPairRevertsOnZeroAddresses() public {
+    function testUpdateDAPairRevertsOnInvalidInput() public {
         vm.startPrank(owner);
 
-        // Both addresses zero
-        vm.expectRevert(ZeroAddress.selector);
-        rollupDAManager.updateDAPair(zeroAddress, zeroAddress, true);
-
         // L1DAValidator zero
         vm.expectRevert(ZeroAddress.selector);
-        rollupDAManager.updateDAPair(zeroAddress, l2DAValidator1, true);
+        rollupDAManager.updateDAPair(zeroAddress, L2DACommitmentScheme.EMPTY_NO_DA, true);
+
+        // L2DACommitmentScheme NONE
+        vm.expectRevert(
+            abi.encodeWithSignature("InvalidL2DACommitmentScheme(uint8)", uint8(L2DACommitmentScheme.NONE))
```

Comment:
can you maybe import the event, just to not have encodeWithSignature. Sorry for the inconveniance

Reply:
Sure. I'll also add some new tests for Admin facet